File: Python Assignments/Python Assignment 3/Assignment3_4.py
#   4. Write a program which accept N numbers from user and store it into
#      List. Accept one another number from user and return frequency
#      of that number from List.

def NumFrequency(Data, No):
    count = 0
    for i in Data:
        if i != No:
            continue
        else:
            count = count + 1
    return count

def main():
    Size = int(input("How many elements do you want in the list : "))
    Arr = []

    print("Enter a list elements : ")
    for i in range(Size):
        No = int(input())
        Arr.append(No)
    print("Your Entered List is : ",Arr)

    Element = int(input(("Which Element of frequecy is to search in your entered list: ")))
    Frequency = NumFrequency(Arr, Element)
    print("Frequency of {0} is {1}".format(Element, Frequency))

if __name__ == "__main__":
    main()

#   Output:
#   D:\Python\Marvellous Python\Python Assignments\Python Assignment 3>Python Assignment3_4.py
#   How many elements do you want in the list : 11
#   Enter a list elements :
#   13
#   5
#   45
#   7
#   4
#   56
#   5
#   34
#   2
#   5
#   65
#   Your Entered List is :  [13, 5, 45, 7, 4, 56, 5, 34, 2, 5, 65]
#   Which Element of frequency is to search in your list : 5
#   Frequency of 5 is 3

File: Logic_Building_Assignments/Logic_Building_Assignment_3/Assignment3_4.c
//  4. Accept one character from user and convert case of that character.

#include<stdio.h>

char DisplayConvert(char cValue)
{
    if((cValue >= 'A') && (cValue <= 'Z'))
    {
        return cValue + 32;
    }
    else if((cValue >= 'a') && (cValue <= 'z'))
    {
        return cValue - 32;
    }
    else
    {
        return '\0';
    }
}

int main()
{
    char cValue = '\0', Character = '\0';

    printf("Enter your character : ");
    scanf("%c",&cValue);

    Character = DisplayConvert(cValue);
    if(Character == '\0')
    {
        printf("%c is not a alphabate",cValue);
    }
    else
    {
        printf("%c",Character);
    }

    return 0;
}

/*

    Output:
    D:\Marvellous Infosystems\Logic Building with Industrial Project Batch\Logic Building Assignments\Logic Building Assignment 3>gcc Assignment3_4.c

    D:\Marvellous Infosystems\Logic Building with Industrial Project Batch\Logic Building Assignments\Logic Building Assignment 3>a
    Enter your character : a
    A
    D:\Marvellous Infosystems\Logic Building with Industrial Project Batch\Logic Building Assignments\Logic Building Assignment 3>a
    Enter your character : D
    d
    D:\Marvellous Infosystems\Logic Building with Industrial Project Batch\Logic Building Assignments\Logic Building Assignment 3>a
    Enter your character : @
    @ is not a alphabate

*/